ts/Dashboards/EditMode/EditMode.ts

```diff
@@ -622,12 +622,31 @@ class EditMode {
                         className: EditGlobals.classNames.editToolsBtn,
                         icon: (rwdIcons as any)[key] || '',
                         value: key,
-                        callback: (): void => {
-                            this.board.layoutsWrapper.style.width =
-                                rwdBreakingPoints[key] + 'px';
-                            this.rwdMode = key;
-
-                            // reflow elements
+                        callback: (e: PointerEvent): void => {
+                            const button = e.target as HTMLElement,
+                                isSelected =
+                                    button.classList.contains('selected');
+
+                            // Deselect given button and reset board width.
+                            if (isSelected) {
+                                button.classList.remove('selected');
+                                this.board.layoutsWrapper.style.width = '100%';
```

Comment:
You can remove the width from the style instead of applying 100%. It is by default in our CSS.